VierGewinntM404/Form2.cs

```diff
@@ -43,9 +43,20 @@ struct player
         public Form2(string PlayerOne, string PlayerTwo) // Form nimmt die Namen der 2 Spieler als Input
         {
             InitializeComponent();
-            Button[,] Field = { { btn1_1, btn1_2, btn1_3, btn1_4, btn1_5 }, { btn2_1, btn2_2, btn2_3, btn2_4, btn2_5 }, { btn3_1, btn3_2, btn3_3, btn3_4, btn3_5 }, { btn4_1, btn4_2, btn4_3, btn4_4, btn4_5 }, { btn5_1, btn5_2, btn5_3, btn5_4, btn5_5 }, { btn6_1, btn6_2, btn6_3, btn6_4, btn6_5 }, { btn7_1, btn7_2, btn7_3, btn7_4, btn7_5 }, { btn8_1, btn8_2, btn8_3, btn8_4, btn8_5 }, { btn9_1, btn9_2, btn9_3, btn9_4, btn9_5 }, { btn10_1, btn10_2, btn10_3, btn10_4, btn10_5 } };
-            /* fülle 2D Button Array mit Buttons*/
```

Comment:
So ein Array zu definieren ist sehr unleserlich in zukunft auf verschiedene Zeilen verschieben